homeassistant/components/esphome/__init__.py

```diff
@@ -149,7 +149,8 @@ def async_update_device_state(self, hass: HomeAssistantType) -> None:
 
 
 def _attr_obj_from_dict(cls, **kwargs):
-    return cls(**{key: kwargs[key] for key in attr.fields_dict(cls)})
+    return cls(**{key: kwargs[key] for key in attr.fields_dict(cls)
```

Comment:
That's why in HA Core, we always write out each field that we expect/use explicitly. We had a few times issues with people not being able to roll back because of it.

Reply:
Yeah I know :) I'm just too lazy for that 😆